modules/vault/pom.xml

```diff
@@ -33,4 +33,24 @@
 
     <artifactId>ignite-vault</artifactId>
     <version>3.0.0-SNAPSHOT</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+        </dependency>
+
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-core</artifactId>
+            <version>3.0.0-SNAPSHOT</version>
```

Comment:
```suggestion
            <version>${project.version}</version>
```

Reply:
Version should be removed completely. See https://issues.apache.org/jira/browse/IGNITE-14630